pom.xml

```diff
@@ -228,6 +228,11 @@
       <version>3.4.1</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>commons-lang</groupId>
+      <artifactId>commons-lang</artifactId>
+      <version>2.4</version>
+    </dependency>
```

Comment:
I don't like adding new dependencies when existing dependencies will do the job.

Reply:
Ah, we can use the Guava Joiner for now and then we switch it out after dropping support for Java 7. Good catch, thanks!